### PR TITLE
项目列表页英文翻译 #189

### PR DESCRIPTION
**Pull request by anndawn**
项目列表页英文翻译 #189

### CODE REVIEW

File: site_config/project.js [hunk 1]
```diff
@@ -6,30 +6,30 @@ export default {
...
+                    title: 'Wuhan New Coronavirus Outbreak Prevention Information Collection Platform',
                     repoLink:'https://github.com/wuhan2020/wuhan2020',
-                    description: "Wuhan new coronavirus epidemic prevention information collection platform",
+                    description: "Wuhan new coronavirus outbreak prevention information collection platform",
                     link: 'https://wh.opensource-service.cn/',
                 },
                 {
                   img: '/images/blog/wuhan2020.png',
-                  title: '2020援助武汉',
+                  title: '2020 Support Wuhan',
```
Comment (GindaChen) [inline site_config/project.js:16]: Hmm... I'm not sure this is the good translation. Something like
```
2020 Wuhan Rescue
```
would sound better

File: site_config/project.js [hunk 2]
```diff
@@ -6,30 +6,30 @@ export default {
             list: [
                 {
                     img: '/images/blog/wuhan2020.png',
-                    title: '新型冠状病毒防疫信息收集平台',
+                    title: 'Wuhan New Coronavirus Outbreak Prevention Information Collection Platform',
```
Comment (GindaChen) [inline site_config/project.js:9]: **Novel**
```
Wuhan Novel Coronovirus Outbreak
```
is the previous name.

File: site_config/project.js [hunk 3]
```diff
@@ -6,30 +6,30 @@ export default {
...
+                    description: "Wuhan new coronavirus outbreak prevention information collection platform",
                     link: 'https://wh.opensource-service.cn/',
                 },
                 {
                   img: '/images/blog/wuhan2020.png',
-                  title: '2020援助武汉',
+                  title: '2020 Support Wuhan',
                   repoLink: 'https://github.com/wuhan2020/data-sync',
-                  description: '建立一个医院、工厂、采购等信息实时同步的数据服务，同时召集所有希望对这次抗击病毒战役进行贡献的人，让每个具有相关技能的人都可以参与相关主题的开发工作，用开源的社区文化，以自组织协作的方式完成。',
+                  description: 'Establish a real-time data service with information on hospitals, factories and procurement. Convene everyone who wishes to contribute to this battle against coronavirus. Allow everyone with relevant skills to participate in the development under relevant topics.',
```
Comment (GindaChen) [inline site_config/project.js:18]: - realtime **data service platform**
- Convene -> Convey? Connect?

File: site_config/project.js [hunk 4]
```diff
@@ -6,30 +6,30 @@ export default {
...
+                  description: 'Wuhan2020 community official website, convene volunteers all over the world from different backgrounds to establish an open-source information system and contribute to battling the 2019-nCoV',
                   link: 'https://community.wuhan2020.org.cn/',
                 },
                 {
                   img: '/images/blog/wuhan2020.png',
-                  title: '2019-nCoV-Nav 疫情信息导航',
+                  title: '2019-nCoV Outbreak Navigation Site',
                   repoLink: 'https://github.com/wuhan2020/2019-nCoV-Nav',
-                  description: '疫情信息导航网站为公众提供一个便利而全面的疫情信息入口，整理了涵盖疫情信息的各个方面的网络通道。',
+                  description: '2019-nCoV navigation site provides the public with a convenient and comprehensive information portal as well as systematically organize all online resources and useful channels related to 2019-nCoV.',
```
Comment (GindaChen) [inline site_config/project.js:32]: Suggest change
```
2019-nCoV navigation site provides the public with a convenient and comprehensive information portal as well as systematically organize all online resources and useful channels related to 2019-nCoV.
```
to
```
2019-nCoV navigation site is a information portal with online resources and archival information related to 2019-nCoV.```

Direct translation will not work here.

File: site_config/project.js [hunk 5]
```diff
@@ -6,30 +6,30 @@ export default {
...
+                  description: 'Establish a real-time data service with information on hospitals, factories and procurement. Convene everyone who wishes to contribute to this battle against coronavirus. Allow everyone with relevant skills to participate in the development under relevant topics.',
                   link: 'https://wuhan2020.kaiyuanshe.cn/',
                 },
                 {
                   img: '/images/blog/wuhan2020.png',
-                  title: 'Wuhan2020开源社区官网',
+                  title: 'Wuhan2020 Open Source Community Official Website',
                   repoLink: 'https://github.com/wuhan2020/wuhan2020.github.io',
-                  description: '目前Wuhan2020社区官方网站，召集不同国家不同职业的志愿者，建立开源信息系统，为抵抗新型冠状病毒做贡献。',
+                  description: 'Wuhan2020 community official website, convene volunteers all over the world from different backgrounds to establish an open-source information system and contribute to battling the 2019-nCoV',
```
Comment (GindaChen) [inline site_config/project.js:25]: Same as above.

**Review by GindaChen — CHANGES_REQUESTED** [review]
Thanks for the contribution! Some direct translation might not work here. It would be great if you can consider tailoring the text for English reader - also other English-as-second-language readers. 
But anyways - thanks for contribution! Love to hear back from you

File: site_config/project.js (hunk 1 above)
Comment (iLtc) [reply site_config/project.js:16]: I personally think "rescue" is too scary.

File: site_config/project.js (hunk 4 above)
Comment (anndawn) [reply site_config/project.js:32]: how about keeping convenient and comprehensive?

File: site_config/project.js (hunk 2 above)
Comment (anndawn) [reply site_config/project.js:9]: √

File: site_config/project.js (hunk 5 above)
Comment (anndawn) [reply site_config/project.js:25]: Wuhan 2020 community calls for volunteers from all over the world to establish an open-source information platform and contribute to battling the 2019-nCoV.
is this okay?

File: site_config/project.js (hunk 3 above)
Comment (anndawn) [reply site_config/project.js:18]: agree on data service platform.  How about this?
Our goal is to establish a real-time data service platform with information on hospitals, factories and procurement, allowing anyone with relevant skills to participate in this initiative. We would like to call for anyone who wishes to contribute to this battle against coronavirus to join us.

File: site_config/project.js (hunk 1 above)
Comment (GindaChen) [reply site_config/project.js:16]: I agree - I just don't know what's better...

File: site_config/project.js (hunk 4 above)
Comment (GindaChen) [reply site_config/project.js:32]: Yep - go for it

File: site_config/project.js (hunk 5 above)
Comment (GindaChen) [reply site_config/project.js:25]: `battling` ... sounds weird. `combat` will be grammatically correct but too political.
`open source platform related to 2019-nCoV` should be okay?

**Pull request merged**